scripts/install-deps.sh

```diff
@@ -11,6 +11,7 @@
 # Ensure that getopt starts from first option if ". <script.sh>" was used.
 OPTIND=1
 
+ret=""
```

Comment:
?

Reply:
It needed to be declared so it was at a larger scope I think?